src/Phoenix.vue

```diff
@@ -39,17 +51,35 @@ export default {
   },
   computed: {
     ...mapState(['route', 'user']),
-    ...mapGetters(['configuration', 'activeNotifications', 'activeMessages', 'capabilities']),
+    ...mapGetters(['configuration', 'activeNotifications', 'activeMessages', 'capabilities', 'apps']),
     $_applicationsList () {
-      return this.configuration.applications
+      const list = []
+
+      // Get extensions manually added into config
+      list.push(this.configuration.applications)
+
+      // Get extensions which have at least one nav item
+      const navItems = this.$root.navItems
+      for (const item in navItems) {
+        list.push(this.apps[item])
+      }
+
+      return list.flat()
     },
 
     appNavigationEntries () {
       if (this.publicPage()) {
         return []
       }
+
+      const items = this.$root.navItems[this.currentExtension]
```

Comment:
aha, it took me a while to find where you do the actual filtering. Here it is :-)

Reply:
Originally wanted to move it somewhere else but since you're touching phoenix.js file and part of it is living in there I wanted to wait when that gets merged to align it together.